xonsh/pyghooks.py

```diff
@@ -360,7 +360,7 @@ def _partial_color_tokenize_main(template, styles):
 
 
 class CompoundColorMap(MutableMapping):
-    """Looks up color tokes by name, potentailly generating the value
+    """Looks up color tokes by name, potentially generating the value
```

Comment:
You mean it's not talking about taking a drag off a cigarette? 😉 

I'll take care of this too!